### PR TITLE
Cleanup SCDF k8s docs

### PR DESCRIPTION
**Pull request by chrisjs**
Resolves https://github.com/spring-cloud/spring-cloud-dataflow/issues/3673

### CODE REVIEW

File: content/documentation/pages/1-installation/3-kubernetes/1-helm.md [hunk 1]
```diff
@@ -40,23 +46,9 @@ You should see the `Tiller` pod running.
...
-| SCDF-K8S-Server 1.4.x |     ❌      |     ✅      |     ❌      |     ❌      |
-| SCDF-K8S-Server 1.5.x |     ❌      |     ✅      |     ❌      |     ❌      |
-| SCDF-K8S-Server 1.6.x |     ❌      |     ✅      |     ❌      |     ❌      |
-| SCDF-K8S-Server 1.7.x |     ❌      |     ❌      |     ✅      |     ❌      |
-| SCDF-K8S-Server 2.0.x |     ❌      |     ❌      |     ❌      |     ✅      |
-
 #### Expected output
 
-You should see the following output:
+After issuing the `helm install` command, you should see output similar to the following:
```
Comment (sabbyanandan) [inline content/documentation/pages/1-installation/3-kubernetes/1-helm.md]: Let's update the NOTE from "Get the application URL by running these commands:" --> "Get the Spring Cloud Data FLow's application URL by running these commands:"

File: content/documentation/pages/1-installation/3-kubernetes/1-helm.md [hunk 2]
```diff
@@ -114,33 +106,27 @@ You can watch the status of the server by running `kubectl get svc -w my-release
...
+<!--END_NOTE-->
+
+You have just created a new release in the default namespace of your Kubernetes cluster.
+It takes a couple of minutes for the application and its required services to start.
+You can check on the status by issuing a `kubectl get pod -w` command.
+You need to wait for the `READY` column to show `1/1` for all pods.
+
+When all pods are ready, you can access the Spring Cloud Data Flow dashboard by accessing `http://<SERVICE_ADDRESS>/dashboard` where `<SERVICE_ADDRESS>` is the address returned by either the `kubectl` or `minikube` commands above.
+
+To see what `Helm` releases you have running, you can use the `helm list` command.
```
Comment (sabbyanandan) [inline content/documentation/pages/1-installation/3-kubernetes/1-helm.md]: > To see what `Helm` releases of Spring Cloud Data Flow you have running, you can use the `helm list` command.

File: content/documentation/pages/1-installation/3-kubernetes/1-helm.md [hunk 3]
```diff
@@ -114,33 +106,27 @@ You can watch the status of the server by running `kubectl get svc -w my-release
...
+When it is time to delete the release, run `helm delete my-release`.
```
Comment (sabbyanandan) [inline content/documentation/pages/1-installation/3-kubernetes/1-helm.md]: > When it is time to delete the previously installed SCDF release, run `helm delete my-release`.

File: content/documentation/pages/1-installation/3-kubernetes/1-helm.md [hunk 4]
```diff
@@ -114,33 +106,27 @@ You can watch the status of the server by running `kubectl get svc -w my-release
...
+This command removes any resources created for the release but keeps release information so that you can roll back any changes by using a `helm rollback my-release 1` command.
```
Comment (sabbyanandan) [inline content/documentation/pages/1-installation/3-kubernetes/1-helm.md]: "rollback"

File: content/documentation/pages/1-installation/3-kubernetes/1-helm.md [hunk 5]
```diff
@@ -160,6 +146,18 @@ helm install --name my-release \
     --set mysql.mysqlRootPassword=mysqlpwd incubator/spring-cloud-data-flow
 ```
 
+#### Version Compatibility
+
+The following listing shows Spring Cloud Data Flow’s Kubernetes version compatibility with the respective Helm Chart releases:
```
Comment (sabbyanandan) [inline content/documentation/pages/1-installation/3-kubernetes/1-helm.md]: > The following listing shows Spring Cloud Data Flow’s version compatibility with the respective Helm Chart releases:

Having Kubernetes in the sentence makes it sound like K8s version, but it is the SCDF version we are referring to here, so let's remove the word.

File: content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md [hunk 6]
```diff
@@ -68,9 +68,21 @@ Now that you have defined a stream, you can deploy it. To do so:
...
-    ![Unique Port](images/dataflow-unique-port.png)
+If you use the local Data Flow Server, add the following deployment property to set the port to avoid a port collision.
+
+![Unique Port](images/dataflow-unique-port.png)
+
+<!--END_NOTE-->
+
+<!--NOTE-->
+
+If you use the Kubernetes Data Flow Server, add the deployment property `kubernetes.createLoadBalancer` with a value of `true` on the `http` source application to expose the service externally.
```
Comment (sabbyanandan) [inline content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md]: There's only one server now anyway, so no need to present it as "Kubernetes Data Flow Server".

File: content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md [hunk 7]
```diff
@@ -162,12 +174,41 @@ cf logs http-ingest-314-log-v1
...
+
+If your using Minikube, you can use the following command to get the URL for the server:
+
+```bash
+export SERVICE_URL=$(minikube service --url test-http-v1)
+```
+
+<!--END_NOTE-->
+
+You can view the address for your platform by typing the following:
```
Comment (sabbyanandan) [inline content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md]: > You can view the HTTP URL of the application by typing the following:

Comment (sabbyanandan) [reply content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md]: Making it more descriptive might help, I think. That's why "HTTP URL"

**Comment by chrisjs** [comment]
comments addressed

**Comment by chrisjs** [comment]
merged in 2d1091adc0f944a4ade5d667b243da4909a7da4e

**Pull request closed without merging**